cdap-docs/developer-manual/source/pipelines/developing-plugins/creating-a-plugin.rst

```diff
@@ -242,12 +242,38 @@ Below is an example of an ``ErrorCollector`` that adds the error stage, code, an
    :language: java
    :lines: 34-
 
+.. _cdap-pipelines-creating-a-plugin-alert-publishers:
+
+.. highlight:: java
+
+Alert Publisher Plugin
+======================
+An ``AlertPublisher`` plugin is a special type of plugin that consumes alerts emitted from previous stages
+instead of output records. Alerts are meant to be uncommon events that need to be acted on in some other
+program. Alerts contain a payload, which is just a map of strings containing any relevant data.
```

Comment:
can there be an example of what to contain in the payload and how the payload can be used?